src/sentry/static/sentry/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -37,14 +231,20 @@ class WidgetCard extends React.Component<Props> {
       >
         <StyledPanel>
           <WidgetHeader>{widget.title}</WidgetHeader>
-          <StyledPanelBody>{JSON.stringify(widget.queries)}</StyledPanelBody>
+          <StyledPanelBody>
+            {widget.queries.map((widgetQuery, index) =>
+              this.renderWidgetQuery(widgetQuery, index)
```

Comment:
If a widget has multiple queries they should all be plotted on the same chart as different series. This looks like it would generate multiple charts inside the widget.

Reply:
Ah I see.